docs/devguide/hexformat.rst

```diff
@@ -67,3 +105,48 @@ Running the ``make all`` command executes the following steps:
 - The ``tools/adduicr.py`` script adds the UICR to the bare hex
 - The final hex file is placed in ``build/firmware.hex``
 - The user can optionally append a script using ``tools/makecombinedhex.py`` (or other tools)
+
+micro:bit V2
+============
+
+This applies to MicroPython for the micro:bit V2, the source of which can be found here: `microbit-foundation/micropython-microbit-v2 <https://github.com/microbit-foundation/micropython-microbit-v2>`_.
+
+This is a port of MicroPython to the micro:bit which uses the CODAL as the underlying target platform.
+
+After cloning this repository update the submodules:
+
+`$ git submodule update --init`
```

Comment:
To get the code block right I think this might be the format:
```suggestion
After cloning this repository update the submodules::

    $ git submodule update --init
```

Otherwise we get:
![image](https://user-images.githubusercontent.com/29712657/106493612-48e6b400-64b1-11eb-85d7-d445ea065715.png)

Reply:
Same applies to the other commands below.